debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java

```diff
@@ -138,6 +145,25 @@ public Long xmin() {
         return this.xmin;
     }
 
+    public String sequence() {
+        StringBuilder sequence = new StringBuilder("[");
+        if (this.sequence != null) {
+            for (int i = 0; i < this.sequence.length; i++) {
+                if (this.sequence[i] == null) {
+                    sequence.append((Object) null);
+                }
+                else {
+                    sequence.append(this.sequence[i].asLong());
```

Comment:
Did not we agree on having the array to be array of Strings?

Reply:
Ah, that's a good point actually; in JSON, we couldn't have an array of mixed types, right? So something like `"[123,\"foo\"]"` wouldn't be doable. So yes, we'd either have to make it all strings, also for values which are actually numbers. Or have a custom syntax which isn't valid JSON necessarily. I suppose the former (all strings) makes it slightly better digestible for consumers. Also that all parts of the array are sortable uniformly (lexicographically, that is), is good.

Reply:
My mistake -- thank you for catching this!